src/main/java/org/mule/extension/api/introspection/AbstractImmutableModel.java

```diff
@@ -109,10 +118,7 @@ public int hashCode()
     @Override
     public String toString()
     {
-        return new StringBuilder().append("AbstractImmutableModel{")
-                .append("name='").append(name).append('\'')
-                .append(", description='").append(description).append('\'')
-                .append(", modelProperties=").append(modelProperties).append('}').toString();
+        return ToStringBuilder.reflectionToString(this);
```

Comment:
Would it make sense to implement this so that modelProperties is represented as its values(), for a more concise representation?